dev-util/codespell/codespell-2.2.1.ebuild

```
@@ -0,0 +1,43 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..11} pypy3 )
+
+inherit distutils-r1
+
+DESCRIPTION="Check text files for common misspellings"
+HOMEPAGE="https://github.com/codespell-project/codespell"
+SRC_URI="https://github.com/codespell-project/codespell/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+# Code licensed under GPL-2
+# Dictionary licensed under CC-BY-SA-3.0
+LICENSE="GPL-2 CC-BY-SA-3.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+BDEPEND="
+	sys-apps/help2man
+	test? ( dev-python/chardet[${PYTHON_USEDEP}] )
+"
+
+distutils_enable_tests --install pytest
```

When moving to PEP517 mode, you will want to change it into (in this mode it always does install):
```suggestion
distutils_enable_tests pytest
```